src/main/java/subway/application/SectionService.java

```diff
@@ -0,0 +1,58 @@
+package subway.application;
+
+import java.util.List;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import subway.application.util.Finder;
+import subway.domain.Line;
+import subway.domain.LineRepository;
+import subway.domain.Section;
+import subway.domain.Station;
+import subway.domain.StationRepository;
+import subway.dto.SectionRegisterRequest;
+import subway.exception.LineNotFoundException;
+import subway.exception.StationNotFoundException;
+
+@Transactional(readOnly = true)
+@Service
+public class SectionService {
+    private final LineRepository lineRepository;
+    private final StationRepository stationRepository;
+    private final Finder finder;
+
+    public SectionService(
+            final LineRepository lineRepository,
+            final StationRepository stationRepository,
+            final Finder finder
+    ) {
+        this.lineRepository = lineRepository;
+        this.stationRepository = stationRepository;
+        this.finder = finder;
+    }
+
+    @Transactional
+    public void registerSection(final Long lineId, final SectionRegisterRequest sectionRegisterRequest) {
+        Line line = lineRepository.findById(lineId)
+                .orElseThrow(LineNotFoundException::new);
+
+        Section section = createSection(sectionRegisterRequest, line);
+
+        line.addSection(section);
```

Comment:
Line 에 Section을 추가 하는 코드를 작성할 떄마다 Section을 생성하는 코드를 작성해 줘야 하는데요,
다음과 같이 Line 에서 Section을 생성하고 추가해 주면 어떨까요? 😄 
```suggestion
        line.addSection(sectionRegisterRequest.getDistance(), upStation, downStation);
```